src/styles/Task.scss

```diff
@@ -2,10 +2,11 @@
   border-radius: 10px;
   display: flex;
   flex-direction: column;
-  margin: 1vh auto;
-  margin-bottom: 5vh;
+  margin: 0 132px;
```

Comment:
Minor: Consider using rem in the future - https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15
This is better for accessibility since this will allow it to better scale with font size

Reply:
Yeah, the 132px is to match the margin of other components. A future refactoring of some of the CSS would likely incorporate this change.